packages/credential-w3c/src/action-handler.ts

```diff
@@ -573,11 +619,17 @@ export class CredentialIssuer implements IAgentPlugin {
           challenge: args.challenge,
           domain: args.domain,
           audience,
+          policies: args.policies
```

Comment:
I think policies should be translated here to match the JWT specific ones expected by did-jwt

Reply:
Yep, understood.